enhancements/kueue/openshift-kueue.md

```diff
@@ -517,14 +650,38 @@ d. AI usage versus Non AI usage.
 
 This is still up in the air but one idea is that we can use telemetry for GPU or RHOAI metrics to determine AI users versus non AI users.
 
-#### Existing Usage of Kueue
+#### RHOAI Integration
+
+RHOAI has an uber operator called OpenDataHub. This operator provides the ability to install
+various upstream projects. 
+RHOAI provides the flexibility to install Kueue and treat it as a managed project. 
+
+RHOAI considers clusterqueues, localqueues, workloads, resourceflavors, and workloadpriorities as a GA API for customers.
 
-RHOAI is a major user of Kueue and they have already released Kueue as GA. 
-The integrations that are GA for RHOAI, as of writing this enhancement, are Kubeflow Training Operator and Ray.
+Customers are able to change the configuration of Kueue but that modification is not persisted upon upgrades.
+They can either patch the configmap or in an advanced use cases they can provide kustomize to the RHOAI operator to
+deploy a custom Kueue installatoin.
+
+The integrations that are GA for RHOAI are Kubeflow Training Operator and Ray.
 
 RHOAI also uses a single release of Kueue across the latest 4 releases of Openshift (ie 4.14, 4.15, 4.16, 4.17).
 A request from them for this operator is supporting Kueue across multiple releases of Openshift.
 
+#### Features that require certain Kubernetes versions
+
+MultiKueue depends on the JobManagedBy feature gate in Kubernetes which is beta in 1.32.
+
+ProvisionACC uses ProvisionRequests which are not yet supported in Openshift. 
+Kueue uses the v1beta1 API and Openshift will support V1 API in 4.19.
+There will be some upstream work needed to have a smarter switch so that 
+we can use the V1 API if it exists.
+
+#### No support of alpha features for kueue
+
+To not break existing users, we will not install Custom Resources 
+that are alpha. This will be filtered out in the operator.
+Only beta APIs and GA APIs will be available for use from Kueue.
```

Comment:
Are you concerned about the users installing the alpha APIs themselves? The Gateway API project are preventing other actors from installing CRDs that they consider part of their ecosystem.

For you, that would mean a VAP preventing writes to any Kueue API once your operator is installed, apart from where the write comes from your operator.

That would give you a way in the future to be confident in introducing other Kueue APIs without worry that users have got the old alpha versions already installed?

Reply:
To be clear, we are not registered the alpha APIS as a Custom Resource in the APIServer. So APIServer will reject those CRs.

I think that is sufficient in this case. We also disable the feature gates for these features as they are alpha.

Reply:
Fair, I think that's a perfectly reasonable way to go, just thought I'd mention the Gateway API example as they went the other way, they're concerned about the upgrade story and want to make sure it's safe to take over management of the CRDs in all OpenShift clusters. As this is an add-on, it is a slightly different approach though

Reply:
I think we can resolve this for now. This may be a case of YANGI. If we need it we can add this but I don't think its necessary to add at this moment.